tests/test_loads.py

```diff
@@ -137,6 +137,10 @@ def test_decode_failure_value(self):
         with self.assertRaises(UnicodeDecodeError):
             self.pghstore.loads(s)
 
+    def test_round_trip_double_quotes(self):
+        d = {'key_"quoted"_string': 'value_"quoted"_string'}
```

Comment:
The failure case I was using had a backslash as a real value.

Reply:
4cf43ec should contain that as a test case.